src/Bridge/Symfony/Bundle/Command/SwaggerCommand.php

```diff
@@ -68,13 +68,14 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $documentation = new Documentation($this->resourceNameCollectionFactory->create(), $this->apiTitle, $this->apiDescription, $this->apiVersion, $this->apiFormats);
         $data = $this->documentationNormalizer->normalize($documentation);
         $content = $input->getOption('yaml') ? Yaml::dump($data, 6, 4, Yaml::DUMP_OBJECT_AS_MAP) : json_encode($data, JSON_PRETTY_PRINT);
-        if (!empty($input->getOption('output'))) {
-            file_put_contents($input->getOption('output'), $content);
+        if (!empty($filename = $input->getOption('output')) && \is_string($filename)) {
+            file_put_contents($filename, $content);
             $output->writeln(
-                sprintf('Data written to %s', $input->getOption('output'))
+                sprintf('Data written to %s', $filename)
             );
         } else {
-            $output->writeln($content);
+            $output->writeln((string) $content);
```

Comment:
why do we have to cast here? Maybe that we should cast above in the variable declaration?

Reply:
that was because of the `json_encode` that couls return false but I've coreted in a "cleaner" way I think